public/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

Comment:
this line should probably be removed now ive added the mobile version below

Reply:
yeah

Reply:
take it away